src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -213,7 +213,6 @@ a.anchor,
 .mobile-topbar h2 a,
 h1 a,
 .search-results a,
-.stab,
```

Comment:
`color: var(--main-color);` is everywhere. It's here and also in the stab definition.
(Actually, I mistakenly removed this line, but let's see what you think)

Reply:
Removing it should be fine :+1: